### PR TITLE
kelseys practice

### PR DESCRIPTION
**Pull request by KelseyBaker262**
Hello, this is my practice branch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@ <h1>Hello There Syntax Squad!</h1>
       <p>We would like you all to tryout GitHub Flow.</p>
       <p>Winnie and Jules 💋</p>
       <p>Welcome Ayo, Kelsey, Jose and Albert</p>
+      <p>Kelsey's practice 🌻</p>
```
Comment (lazycloud0) [inline index.html:14]: Cool! you might want to try change up something and tick off comments. I'll request change.

**Review by lazycloud0 — CHANGES_REQUESTED** [review]
Try change it up and tick off the comment.

File: index.html (hunk 1 above)
Comment (josetutis) [reply index.html:14]: Hi Kelsey can you add me as a reviewer so I can practise that side of the flow please :)

Comment (KelseyBaker262) [reply index.html:14]: Hey @josetutis I think I've made you a reviewer now, any issues let me know :)

**Review by lazycloud0 — APPROVED** [review]
Is 🌻 the change?

**Review by julesgsmith — APPROVED** [review]
Approved from here :)

**Pull request merged**